From a621f63733258e3aaca4bf43acb127eddfe75633 Mon Sep 17 00:00:00 2001
From: Linu Cherian <lcherian@marvell.com>
Date: Thu, 2 Apr 2020 17:11:34 +0530
Subject: [PATCH 0380/1921] octeontx2-af: Add new silicon variant Loki

Add Loki to the list of silicons that are post 96xx C0.

Change-Id: I893b48b51c9d22d075fb0692562fc8d9688ca5c8
Signed-off-by: Linu Cherian <lcherian@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/26232
Reviewed-on: https://sj1git1.cavium.com/26235
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu.h b/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
index d69cc2344ec9..7958d83bea12 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
@@ -510,7 +510,8 @@ static inline bool is_rvu_post_96xx_C0(struct rvu *rvu)
 {
 	struct pci_dev *pdev = rvu->pdev;
 
-	return (pdev->revision == 0x08) || (pdev->revision == 0x30);
+	return (pdev->revision == 0x08) || (pdev->revision == 0x30) ||
+		(pdev->revision == 0x20);
 }
 
 static inline bool is_rvu_96xx_A0(struct rvu *rvu)
-- 
2.31.1

